project/src/components/review-form/review-form.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState, ChangeEvent } from 'react';
+
+
+function ReviewForm(): JSX.Element {
+
+  const [formData, setFormData] = useState({
+    rating: 1,
+    reviewText: '',
+
+  });
+
+  const fieldChangeHandle = (evt: ChangeEvent<HTMLInputElement> | ChangeEvent<HTMLTextAreaElement>) => {
+    const { name, value } = evt.target;
+    setFormData({ ...formData, [name]: value });
+  };
+
+  const ratings = Array.from({ length: 10 }, (v, i) => 1 + i).reverse();
```

Comment:
```suggestion
  const ratings = Array.from({ length: 10 }, (_, i) => 1 + i).reverse();
```
Нижнее подчеркивание, чтобы подчеркнуть, что не используется